README.md

```diff
@@ -1,8 +1,42 @@
 # BitClout Frontend
 This is the repository for the reference BitClout Angular app. It serves as an
 interface that allows users to interact with the data on the BitClout blockchain.
+## Getting started
+Bitclout frontend-ui depends on node Node.js, currently tested on v14.16.1.
+
+On MacOs  and Unbuntu, the most reliable way to install Node is to use the [Node Version Manager](https://github.com/nvm-sh/nvm) which allows you to have multiple node versions installed at once just incase you have other applications that depends on different versions.
```

Comment:
```suggestion
On MacOS and Linux based systems, the most reliable way to install Node is to use the [Node Version Manager](https://github.com/nvm-sh/nvm) which allows you to have multiple node versions installed at once just incase you have other applications that depends on different versions.
```

Reply:
thanks for the suggestions added windows node installation link..